mayastor-test/csi-e2e/driver/mayastor_driver.go

```diff
@@ -0,0 +1,61 @@
+package driver
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+//	"github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+//	"k8s.io/apimachinery/pkg/api/resource"
+//	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+)
+
+// MayastorDriverNameVar is the environment variable use to switch the driver to be used.
+const MayastorDriverNameVar = "MAYASTOR_CSI_DRIVER"
+
+// MayastorDriver implements DynamicPVTestDriver interface
+type MayastorDriver struct {
+	driverName string
+}
+
+// InitMayastorDriver returns MayastorDriver that implements DynamicPVTestDriver interface
+func InitMayastorDriver() PVTestDriver {
+	driverName := os.Getenv(MayastorDriverNameVar)
+	if driverName == "" {
+		driverName = "io.openebs.csi-mayastor"
+	}
+
+	klog.Infof("Using Mayastor driver: %s", driverName)
+	return &MayastorDriver{
+		driverName: driverName,
+	}
+}
+
+// normalizeProvisioner extracts any '/' character in the provisioner name to '-'.
```

Comment:
"Replaces..with" instead of "extracts..to".

Reply:
Done.